oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -326,8 +332,23 @@ public APIGatewayProxyResponseEvent handleRequest(
                                 persistentSessionId);
 
                 var accountStatus =
-                        accountInterventionService.getAccountStatus(
-                                userInfo.getSubject().getValue(), auditContext);
+                        configurationService.isAccountInterventionServiceCallEnabled()
+                                ? accountInterventionService.getAccountStatus(
+                                        userInfo.getSubject().getValue(), auditContext)
+                                : null;
+
+                if (nonNull(accountStatus)) {
+                    if (accountStatus.blocked()
+                            || accountStatus.resetPassword()
+                            || (!identityRequired && accountStatus.suspended())) {
+                        return logoutService.handleAccountInterventionLogout(
```

Comment:
Should there be the check for the AISActionEnabled here as logging out is an action?

Reply:
I persisted the logic from https://github.com/govuk-one-login/authentication-api/pull/3698 but I think you're right, have changed